test/integration.sh

```diff
@@ -77,10 +91,45 @@ if ./tmp/bin/up \
   --labels='_id="test"' \
   --token="$token"; then
   result=0
-  printf "\t## tests: ok"
+  printf "\t## metrics tests: ok\n\n"
+fi
+
+echo "-------------------------------------------"
+echo "- Logs Write tests                        -"
+echo "-------------------------------------------"
+
+result=1
+if curl \
+     -v \
+     -H "Content-Type: application/json" \
+     --cacert ./tmp/certs/ca.pem \
+     --cert ./tmp/certs/client.pem \
+     --key ./tmp/certs/client.key \
+     -XPOST \
+     -s "https://127.0.0.1:8443/api/logs/v1/api/v1/push" \
+     --data-raw \
+     '{"streams": [{ "stream": { "foo": "bar2" }, "values": [ [ "1570818238000000000", "fizzbuzz" ] ] }]}'; then
+  result=0
```

Comment:
this new logic currently introduces some potential for danger. If the metrics test fails (result is never set to 0) but the logs test succeeds (result is set to 0 later), then this entire script exits 0. We need to either exit after each failed test, or find another way to combine errors after all tests have run

Reply:
I suggest we go for a fail fast approach.

Reply:
cool that simplifies things  :)